src/components/TextListItem.tsx

```diff
@@ -65,7 +65,7 @@ export const TextListItem: NamedExoticComponent<Props> & {
       disabled={!navigation}
       delayPressIn={0}
       Component={Touchable}
-      accessibilityLabel={`${title} (Taste)`}
+      accessibilityLabel={`(${title})${consts.a11yLabel.button}`}
```

Comment:
I think it is better to have a space between the title and the a11y const. What do you think?

Either way, I suggest keeping it consistent.

Reply:
solved f90ab7f07ff605af93121f7569894af606fd71e3

Reply:
that commit does not change that file at all, and neither does it change the other files to make it consistent.

Reply:
now solved f90ab7f